server/utils/syncModels.js

```diff
@@ -0,0 +1,16 @@
+
+
+
+async function syncModels(sequelize){
+    try{
+        await sequelize.sync();
+        console.log("models synced to DB")
+    }
+    catch(error){
+        console.log(error);
+    }
+};
+
+//sync all models to db (create tables if they dont already exist)
```

Comment:
User.sync() creates table if it doesn't exist (and does nothing if it already exists), is this the reason why I did not see the table on my PgAdmin
![pgadmin4](https://user-images.githubusercontent.com/49102361/93126818-db212380-f69a-11ea-9d7b-b33857321c22.PNG)
?